src/main/java/seedu/ibook/logic/parser/IBookParser.java

```diff
@@ -51,12 +50,12 @@ public Command parseCommand(String userInput) throws ParseException {
         case UpdateCommand.COMMAND_WORD:
             return new UpdateCommandParser().parse(arguments);
 
-        case FindCommand.COMMAND_WORD:
-            return new FindCommandParser().parse(arguments);
-
         case ListCommand.COMMAND_WORD:
-            return new ListCommand();
-
+            if (arguments == null) {
```

Comment:
I think an empty string for arguments is not equivalent to null
```suggestion
            if (arguments.isEmpty()) {
```